app/dao/service_whitelist_dao.py

```diff
@@ -13,5 +14,5 @@ def dao_add_and_commit_whitelisted_contacts(objs):
 
 
 def dao_remove_service_whitelist(service_id):
-    return ServiceWhitelist.query.filter(
-        ServiceWhitelist.service_id == service_id).delete()
+    stmt = delete(ServiceWhitelist).where(ServiceWhitelist.service_id == service_id)
+    db.session.execute(stmt)
```

Comment:
commit

Reply:
Don't commit. The code that uses it is expecting it not to be commit here.
![image](https://github.com/department-of-veterans-affairs/notification-api/assets/16893311/b5d9426d-8f7d-447c-95c5-3e8b48c1911b)